src/geometry/CTiglRelativelyPositionedComponent.cpp

```diff
@@ -112,10 +112,22 @@ void CTiglRelativelyPositionedComponent::SetSymmetryAxis(const TiglSymmetryAxis&
 
 CTiglTransformation CTiglRelativelyPositionedComponent::GetTransformationMatrix() const
 {
-    if (_transformation)
-        return _transformation->getTransformationMatrix();
-    else
-        return CTiglTransformation();
+    const CTiglTransformation thisTransformation = _transformation ? _transformation->getTransformationMatrix() : CTiglTransformation();
+    // TODO(bgruber): this is currently disabled, as it breaks existing behavior
```

Comment:
I don't really like to commit out-commented code.

Reply:
ok, but this will come with a subsequent pull request after we have decided on how parent transformations influence child transformations. any updates on this btw?

Reply:
This is what I found in the CPACS documentation:
  >Transformation type, containing a set of transformations. The order of the transformations is scaling -> rotation -> translation, and they are executed in this order. Any of them can be omitted; it will be replaced by its defaults.Transformations are always executed relative to the child not the parent. I.e. a scaling does not have an influence on the parent item. For example in the outer geometry of a wing the element scaling does not influence the section. Scaling does also not effect rotation and translation.

I don't quite understand the sentence "Transformations are always executed relative to the child not the parent." But the following explanation seems to be clear.